### DISCUSSION TITLE
TBC Classic Port - Beta

### DISCUSSION THREAD

**Discussion by kaytotes** [kaytotes/ImprovedBlizzardUI#175] [General]
Running round for 5 minutes in the BE starting area things seem okay with these changes. This likely needs testing in TBC, Vanilla and Shadowlands.

<hr /><em>This discussion was created from the release <a href='https://github.com/kaytotes/ImprovedBlizzardUI/releases/tag/3.4.0-beta'>TBC Classic Port - Beta</a>.</em>

**Comment by gitgot-sucka** [comment]
My focus frame is still appearing as the default UI.    I don't see an option to enable/disable, so I would assume it's on by default.

**Reply by kaytotes** [reply]
Thank you for pointing that out, I forgot that the focus frame came in with TBC as it was absent in Vanilla. I'll look at adding this asap.

**Comment by Taqito** [comment]
The bug I had that hide my default backpack was fixed.  It would only hide the main bag icon not my other 4 bags that can be replaced.  I believe the issue had something to do with the way bartender and this addon work because when I disabled bartender my bags would show up fine.  Perhaps making an option to disable whatever this addon does to backpacks/bags would be nice.

**Reply by kaytotes** [reply]
I still have no plans to support Bartender.

**Reply by Taqito** [reply]
Whatever you did Its working with bartender now.

**Reply by kaytotes** [reply]
I did nothing related to this. If it's working now I'd say that's bartenders fault.

**Comment by Brich1533** [comment]
I just installed this addon for TBC everything looks good aside from the chat modifications made in retail like font and what not doesn't look to be the same. Plus the green health bar behind my characters portrait extends a little too far down and sticks out just below it underneath the mana bar. Last is it bugs out my bags and keyring, they're completely invisible. I've attached what my UI looks in game so you can see the issues. One other thing my character's portrait frame doesn't auto hide after leaving combat. 

![Improved Blizzard UI Bugs](https://user-images.githubusercontent.com/65371059/119912648-ea46b980-bf29-11eb-8391-254cb575c94a.PNG)

**Reply by Brich1533** [reply]
Disregard this, i just reinstalled the classic edition version and pushed it into my TBC folder and it works/looks like the retail version now. Plus the issues I circled in the screenshot are working as well.

**Reply by kaytotes** [reply]
You should be able to use the standard Improved Blizzard UI which works across Retail, Classic and (hopefully) TBC.

Check that the ImprovedBlizzardUI.toc file contains `## Interface: 90005` if not you need to swap.

**Reply by kaytotes** [reply]
I also see you have a lot of other addons. Please disable those when testing / reporting issues.

**Comment by kaytotes** [comment]
@gitgot-sucka @Brich1533 @Taqito 

I believe all of these issues have been resolved. I am waiting for wowace.com to come back online so I can actually generate a build. Expect one available on CurseForge and WoWInterface as of tomorrow tagged as version 3.4.0.

Thank you for your help testing this as I simply didn't have much time.

I will now close this discussion.
